### DISCUSSION TITLE
[6 settembre 2023] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#16] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/dd45a7c8-614d-4f23-acd4-0bee82d96d22)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/aac5bb68-d8c5-4921-bbba-8ca27a5357c7)
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/5f08ea47-b5ae-43fb-8691-c127a9d5723d)

----


La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Reply by Antaarees** [reply]
Avviso che il diagramma di transizione degli stati del DFA è stato corretto da @Exyss nei suoi appunti.

**Reply by Exyss** [reply]
Ho cambiato l'immagine, grazie del reminder

**Comment by daromacarlo** [comment]
<img width="746" height="818" alt="immagine" src="https://github.com/user-attachments/assets/25537acb-7896-4de2-bd59-7a4d246c9b8c" />

**Reply by daromacarlo** [reply]
<img width="1089" height="753" alt="Immagine 2026-01-05 161258" src="https://github.com/user-attachments/assets/a089c983-66a8-4da5-a789-63d673a79df0" />
<img width="1088" height="779" alt="Immagine 2026-01-05 161311" src="https://github.com/user-attachments/assets/1104c235-2167-468c-865b-5699b57a6085" />
